packages/node/test/unit/fixture/override-merging/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "unused-dependency-override",
+  "name": "override-merging",
   "version": "1.0.0",
   "type": "module",
   "license": "ISC",
   "private": true,
   "main": "index.js",
   "scripts": {
-    "start": "npx snapshot-fs . ../json/unused-dependency-override.json"
+    "start": "npx snapshot-fs ../../json-fixture/override-merging.json"
```

Comment:
BTW, I tried to extract a snapshot-fs json into a folder and the instructions in the repository are outdated.

Reply:
Which repository?

Reply:
...oh, there.